### ISSUE TITLE
Building vnc server fails for Xorg-1.19

### ISSUE THREAD

**Issue by joakim-tjernlund** [TigerVNC/tigervnc#391]
Trying to build tigervnc git  master fails for me, any ideas?

```
libtool: link: x86_64-pc-linux-gnu-ar cru .libs/libvnccommon.a .libs/libvnccommon_la-vncExt.o .libs/libvnccommon_la-vncExtInit.o .libs/libvnccommon_la-vncHooks.o .libs/libvnccommon_la-vncSelection.o .libs/libvnccommon_la-vncBlockHandler.o .libs/libvnccommon_la-XorgGlue.o .libs/libvnccommon_la-RFBGlue.o .libs/libvnccommon_la-XserverDesktop.o .libs/libvnccommon_la-Input.o .libs/libvnccommon_la-InputXKB.o 
libtool: link: x86_64-pc-linux-gnu-ranlib .libs/libvnccommon.a
libtool: link: ( cd ".libs" && rm -f "libvnccommon.la" && ln -s "../libvnccommon.la" "libvnccommon.la" )
libtool: link: x86_64-pc-linux-gnu-g++  -fPIC -DPIC -shared -nostdlib /usr/lib/gcc/x86_64-pc-linux-gnu/4.9.4/../../../../lib64/crti.o /usr/lib/gcc/x86_64-pc-linux-gnu/4.9.4/crtbeginS.o  .libs/libvnc_la-vncModule.o  -Wl,--whole-archive ./.libs/libvnccommon.a -Wl,--no-whole-archive  -Wl,--as-needed 
../../../../common/network/.libs/libnetwork.a ../../../../common/rfb/.libs/librfb.a -L/usr/lib64 -ljpeg -lpam ../../../../common/rdr/.libs/librdr.a -lz -lgnutls ../../../../common/Xregion/.libs/libXregion.a ../../../../common/os/.libs/libos.a -lpthread -L/usr/lib/gcc/x86_64-pc-linux-gnu/4.9.4 -L/usr/lib/gcc/x86_64-pc-linux-gnu/4.9.4/../../../../lib64 -L/lib/../lib64 -L/usr/lib/../lib64 -L/usr/lib/gcc/x86_64-pc-linux-gnu/4.9.4/../../../../x86_64-pc-linux-gnu/lib -L/usr/lib/gcc/x86_64-pc-linux-gnu/4.9.4/../../.. -lstdc++ -lm -lc -lgcc_s /usr/lib/gcc/x86_64-pc-linux-gnu/4.9.4/crtendS.o /usr/lib/gcc/x86_64-pc-linux-gnu/4.9.4/../../../../lib64/crtn.o  -O2 -Wl,-z -Wl,now -Wl,-O1   -Wl,-soname -Wl,libvnc.so -o .libs/libvnc.so
libtool: link: ( cd ".libs" && rm -f "libvnc.la" && ln -s "../libvnc.la" "libvnc.la" )
libtool: link: x86_64-pc-linux-gnu-g++ -O2 -pipe -Wl,-O1 -o Xvnc Xvnc-xvnc.o Xvnc-stubs.o Xvnc-miinitext.o Xvnc-fbcmap_mi.o Xvnc-buildtime.o -Wl,--export-dynamic  -Wl,--as-needed ../../fb/.libs/libfb.a ../../xfixes/.libs/libxfixes.a ../../Xext/.libs/libXext.a ../../dbe/.libs/libdbe.a ../../glx/.libs/libglx.a ../../randr/.libs/librandr.a ../../render/.libs/librender.a ../../damageext/.libs/libdamageext.a ../../miext/sync/.libs/libsync.a ../../miext/damage/.libs/libdamage.a ../../miext/shadow/.libs/libshadow.a ../../Xi/.libs/libXi.a ../../xkb/.libs/libxkb.a ../../xkb/.libs/libxkbstubs.a ../../composite/.libs/libcomposite.a ../../dix/.libs/libmain.a ./.libs/libvnccommon.a ../../../../common/network/.libs/libnetwork.a ../../../../common/rfb/.libs/librfb.a -L/usr/lib64 -ljpeg -lpam ../../../../common/rdr/.libs/librdr.a -lz -lgnutls ../../../../common/Xregion/.libs/libXregion.a ../../../../common/os/.libs/libos.a -lpthread ../../dix/.libs/libdix.a ../../mi/.libs/libmi.a ../../os/.libs/libos.a -lnettle -ldl -lpixman-1 -lXfont2 -lXau -lxshmfence -lXdmcp -lGL -lX11 -lm -pthread
./.libs/libvnccommon.a(libvnccommon_la-vncSelection.o): In function `vncProcConvertSelection':
vncSelection.c:(.text+0x7fc): undefined reference to `ChangeWindowProperty'
vncSelection.c:(.text+0x886): undefined reference to `ChangeWindowProperty'
vncSelection.c:(.text+0x8d3): undefined reference to `ChangeWindowProperty'
vncSelection.c:(.text+0x912): undefined reference to `ChangeWindowProperty'
collect2: error: ld returned 1 exit status
Makefile:818: recipe for target 'Xvnc' failed
make: *** [Xvnc] Error 1
```

**Comment by joakim-tjernlund** [comment]
Seems like ChangeWindowProperty is gone now

**Comment by joakim-tjernlund** [comment]
sed -i 's/ChangeWindowProperty(/dixChangeWindowProperty(serverClient, /' "${WORKDIR}"/tigervnc-9999/unix/xserver/hw/vnc/vncSelection.c
makes it build

**Comment by CendioOssman** [comment]
Good find. I'll try to sort out a patch.

**Comment by CendioOssman** [comment]
Should be fixed in 14f3fc0b83ee516d9631f4e41e3451ddc62e0024.
